### ISSUE TITLE
Feature Request: Improved Accessibility On Bridge Website

### ISSUE THREAD

**Issue by stevenmalenz** [base-org/web#439]
### Is your feature request related to a problem? Please describe.

Bridge is probably one of the most visited pages and while the overall accessibility is good, but I've spotted 5 things that can be improved to make the experience even better for folk with the lived experience of a disability. 

Currently around 1 in 5 have the lived experience of a disability so improvements here could make a big impact to the ecosystem as a whole.

[Here's a Loom video explaining the current problems](https://www.loom.com/share/51a5a0a385314ef684a90ed64d16d5ab)

[@carlaiau](https://github.com/carlaiau) and I are keen to help improve this!

### Form Fields & Color Contrast

The main thing that could heavily improve the accessibility on the Bridge website is in this area.

![Xnapper-2024-04-14-16 21 53](https://github.com/base-org/web/assets/166795968/94f40141-dcfb-4136-a357-8cdd26cf7dd7)

The 'amount' field is missing a label so users relying on screen readers won’t know what to do with the field.

![Xnapper-2024-04-14-16 04 12](https://github.com/base-org/web/assets/166795968/261a2915-c1b4-4f48-a271-1844caafdf04)

There are a lot of elements that have very low contrast which makes it harder for low vision, and colorblind folk to read and use. It also adds eye-strain to all users. 

![Xnapper-2024-04-14-16 04 29](https://github.com/base-org/web/assets/166795968/47e4ee3d-b734-43f1-a6a5-a72eefc19fc7)

Consider at least making the selected chain (for example 'ETH') the same bright white as the other important information.

### Empty Links

The Base logo is an empty link. This is the same on the Base homepage too, but is tagged correctly on Docs.

<img width="346" alt="Xnapper-2024-04-14-16 05 20" src="https://github.com/base-org/web/assets/166795968/e462ad4b-b30d-488e-bfce-0b7a8b4144ff">
<img width="497" alt="Xnapper-2024-04-14-16 24 59" src="https://github.com/base-org/web/assets/166795968/9c75399c-2cca-471a-83c2-5fb996d6d50b">

### Language Element

The language element is missing across the whole site making it harder for screen readers to read content in the appropriate language. 

![Xnapper-2024-04-14-16 23 47](https://github.com/base-org/web/assets/166795968/2fcef6d9-053b-48be-a820-a588a3525606)

### Information Hierarchy

There is no H1 title tag on this website. The Bridge heading is an H2, skipping H1 entirely, so this should be swapped to an H1 tag.

![Xnapper-2024-04-14-16 04 29](https://github.com/base-org/web/assets/166795968/b9e58b54-df2a-4241-ac05-ab30fc710ff3)

### Color Contrast

Finally, the color contrast of the FAQ section is too low according to the WCAG AA standard.

![Xnapper-2024-04-14-16 06 21](https://github.com/base-org/web/assets/166795968/04c051f0-0950-4641-b599-4c7354524da3)

**Comment by stevenmalenz** [comment]
Closed via #448 🎉
